cli/src/cli.rs

```diff
@@ -2187,6 +2165,7 @@ pub fn app<'ab, 'v>(name: &str, about: &'ab str, version: &'v str) -> App<'ab, '
         .subcommand(
             SubCommand::with_name("transfer")
                 .about("Transfer funds between system accounts")
+                .visible_alias("pay")
```

Comment:
Let's hide the alias.  We want to get people off this `pay`

Reply:
Yes, I prefer just `.alias()` as well

Reply:
Done!